docs/toolbox_gendoc.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
```

Comment:
nice :)

should this command be called from `.github/workflows/sphinx-documentation_{test,deploy}.yml`  and appended at the end of `README.rst`?

Reply:
Possibly, but this way we'll end up with an ever-growing README.rst. It should probably be dumped into its own file rather than inside README

Reply:
that's why I was suggesting to scrape only the synopsis, to keep the length reasonable and allow users to quickly discover the available features